<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAB_MainWindow</class>
 <widget class="QMainWindow" name="DAB_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAB_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelAdresse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse : </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAdresse">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelEtat">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditEtat">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnexion">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>491</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Message De La Banque : </string>
    </property>
    <widget class="QPushButton" name="pushButtonNumeroCompte">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro du compte</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNumeroCompte">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Montant : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMontant">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSolde">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Solde</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRetrait">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retrait</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDepot">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dépot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEnvoi">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoi</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
